src/streamlink/plugins/zattoo.py

```diff
@@ -157,13 +160,19 @@ def _hello(self):
         self.session.http.cookies = cookiejar_from_dict({})
         if self.base_url == 'https://zattoo.com':
             app_token_url = 'https://zattoo.com/int/'
+        elif self.base_url == 'https://www.quantum-tv.com':
+            app_token_url = 'https://www.quantum-tv.com/token-4d0d61d4ce0bf8d9982171f349d19f34.json'
         else:
             app_token_url = self.base_url
         res = self.session.http.get(app_token_url)
         match = self._app_token_re.search(res.text)
 
-        app_token = match.group(1)
-        hello_url = self.API_HELLO.format(self.base_url)
+        if self.base_url == 'https://www.quantum-tv.com':
+            app_token = json.loads(res.text)["session_token"]
```

Comment:
you don't need **json.loads**

use **self.session.http.json** or **parse_json**

https://github.com/streamlink/streamlink/blob/6ec0ab028baaae23cb2eb803f05ed26a6b3934c2/src/streamlink/plugins/zattoo.py#L306
https://github.com/streamlink/streamlink/blob/6ec0ab028baaae23cb2eb803f05ed26a6b3934c2/src/streamlink/plugin/api/http_session.py#L100-L106

https://github.com/streamlink/streamlink/blob/6ec0ab028baaae23cb2eb803f05ed26a6b3934c2/src/streamlink/utils/__init__.py#L48-L67